### ISSUE TITLE
Set selected any item programatically

### ISSUE THREAD

**Issue by rpandey91** [Ashok-Varma/BottomNavigation#65]
Is there any way by which I can set selected any item of bottom navigation. Please help

**Comment by Ashok-Varma** [comment]
Use can call selectTab method on your bottomNavigationBar instance with position.
```java
                bottomNavigationBar.selectTab(2);// select tab and call listener
                bottomNavigationBar.selectTab(2, true);// this method selects tab and gives control over listener call
```

**Comment by rpandey91** [comment]
Thanks bhai :-)


Get Outlook for Android






On Sat, Aug 27, 2016 at 7:33 PM +0530, "Ashok Varma" <notifications@github.com> wrote:





Closed #65.

--
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub:
https://github.com/Ashok-Varma/BottomNavigation/issues/65#event-769744358
